spec.md

```diff
@@ -401,9 +401,41 @@ it is fine if the location is detected on forwarding etc.
     </kml>
 
 
-# Miscellaneous
+# Stickers
 
-Messengers SHOULD use the header `In-Reply-To` as usual.
+Stickers are send as normal images
+with the additional header `Chat-Content: sticker`.
+
+It is discouraged to send stickers together with user generated text,
+however, stickers can be used as a reply to a message
+and also the footer should be set as usual.
```

Comment:
Just to check: `SHOULD` or `should`?

Reply:
maybe it is even a MUST - otherwise, the status of a peer would be reset.